manim/mobject/geometry/shape_matchers.py

```diff
@@ -122,6 +122,7 @@ def __init__(
             buff=buff,
             **kwargs,
         )
+        self.color: ManimColor
```

Comment:
Don't worry about this `shape_matchers` issue: it was introduced in a different PR, and it would be solved in another one.
```suggestion
```